### DISCUSSION TITLE
Fix Compression Artifacts?

### DISCUSSION THREAD

**Discussion by JTG-gaming** [codetheweb/muse#888] [Q&A]
I was wondering if there's a way to fix artifacting and cutouts on the music playing. My upload isn't close to being saturated and none of the computers resources are close to full either. Anything to do about this? or is this just a fact of self-hosting. (Running on Windows)

**Comment by codetheweb** [comment]
This is not expected, and if it's happening there's a bottleneck somewhere. I would need to know more about your hosting situation to help further (are you using Docker, how are you monitoring resources, etc.).

I suspect if you try a similar setup with another self-hosted music bot you'll have the same issue.

**Reply by JTG-gaming** [reply]
its being hosted on a dual Xeon x5675 system running windows 10 pro under Node 16.16 LTS
I don't really have a way of monitoring my entire network throughput at the moment, but I cant imagine I'm fully saturating 25mbps of upload bandwidth.
